source/install/install-kubernetes-mattermost.rst

```diff
@@ -80,30 +188,40 @@ The Mattermost installation manifest contains fields which must be edited in lin
     "spec.elasticSearch.username", "The username for your Elasticsearch instance.", "No"
     "spec.elasticSearch.password", "The password for your Elasticsearch instance.", "No"
 
-There are more advanced fields documented `here <https://raw.githubusercontent.com/mattermost/mattermost-operator/master/docs/examples/full.yaml>`__.
-
-**4. Apply the Installation Manifest File**
-
-To initiate deployment, apply the file, specifying the correct path, using:
-
-.. code-block:: sh
-
-  $ kubectl create ns mattermost
-  $ kubectl apply -n mattermost -f /path/to/mattermost-installation.yaml
+7. Create a restore manifest:
 
-The deployment process can be monitored in the Kubernetes user interface.
+Open a text editor and create a text file with the following details. Save the file as ``restore.yaml``:
 
-**4. Configure DNS and Use Mattermost**
+.. code-block:: yaml
 
-When the deployment is complete, obtain the hostname or IP address of your Mattermost deployment using the following command:
+  apiVersion: mattermost.com/v1alpha1
+  kind: MattermostRestoreDB
+  metadata:
+    name: example-mattermostrestoredb
+  spec:
+    initBucketURL: s3://my-sample/my-backup.gz
+    mattermostClusterName: example-clusterinstallation
+    mattermostDBName: mattermostdb
+    mattermostDBPassword: supersecure
+    mattermostDBUser: mmuser
+    restoreSecret: myawscreds
+
+**Parameters**
```

Comment:
If we do choose to include this, should we remove the existing text in the example or keep it in?

Reply:
0/5, might be good to remove it